diff --git a/node_modules/@types/jsdom/base.d.ts b/node_modules/@types/jsdom/base.d.ts
index 420870f..25ac10c 100644
--- a/node_modules/@types/jsdom/base.d.ts
+++ b/node_modules/@types/jsdom/base.d.ts
@@ -196,7 +196,9 @@ declare module "jsdom" {
 
         /* ECMAScript Globals */
         globalThis: DOMWindow;
+        // @ts-ignore
         readonly ["Infinity"]: number;
+        // @ts-ignore
         readonly ["NaN"]: number;
         readonly undefined: undefined;
 
