diff --git a/node_modules/sortablejs/Sortable.js b/node_modules/sortablejs/Sortable.js
index 4582752..14fad4b 100644
--- a/node_modules/sortablejs/Sortable.js
+++ b/node_modules/sortablejs/Sortable.js
@@ -1450,6 +1450,10 @@
         var parent = target;
         while (target && target.shadowRoot) {
           target = target.shadowRoot.elementFromPoint(touchEvt.clientX, touchEvt.clientY);
+          const host = target.getRootNode().host;
+          if (host) {
+            target = host;
+          }
           if (target === parent) break;
           parent = target;
         }
diff --git a/node_modules/sortablejs/modular/sortable.complete.esm.js b/node_modules/sortablejs/modular/sortable.complete.esm.js
index 2cd2ddc..2ac4bb5 100644
--- a/node_modules/sortablejs/modular/sortable.complete.esm.js
+++ b/node_modules/sortablejs/modular/sortable.complete.esm.js
@@ -1444,6 +1444,10 @@ Sortable.prototype = /** @lends Sortable.prototype */{
       var parent = target;
       while (target && target.shadowRoot) {
         target = target.shadowRoot.elementFromPoint(touchEvt.clientX, touchEvt.clientY);
+        const host = target.getRootNode().host;
+        if (host) {
+          target = host;
+        }
         if (target === parent) break;
         parent = target;
       }
diff --git a/node_modules/sortablejs/modular/sortable.core.esm.js b/node_modules/sortablejs/modular/sortable.core.esm.js
index 40aa9fc..d0261fd 100644
--- a/node_modules/sortablejs/modular/sortable.core.esm.js
+++ b/node_modules/sortablejs/modular/sortable.core.esm.js
@@ -1444,6 +1444,10 @@ Sortable.prototype = /** @lends Sortable.prototype */{
       var parent = target;
       while (target && target.shadowRoot) {
         target = target.shadowRoot.elementFromPoint(touchEvt.clientX, touchEvt.clientY);
+        const host = target.getRootNode().host;
+        if (host) {
+          target = host;
+        }
         if (target === parent) break;
         parent = target;
       }
diff --git a/node_modules/sortablejs/modular/sortable.esm.js b/node_modules/sortablejs/modular/sortable.esm.js
index 61f460e..e49b7c2 100644
--- a/node_modules/sortablejs/modular/sortable.esm.js
+++ b/node_modules/sortablejs/modular/sortable.esm.js
@@ -1444,6 +1444,10 @@ Sortable.prototype = /** @lends Sortable.prototype */{
       var parent = target;
       while (target && target.shadowRoot) {
         target = target.shadowRoot.elementFromPoint(touchEvt.clientX, touchEvt.clientY);
+        const host = target.getRootNode().host;
+        if (host) {
+          target = host;
+        }
         if (target === parent) break;
         parent = target;
       }
